ui/src/telegrafs/actions/index.ts

```diff
@@ -1,8 +1,10 @@
 // API
 import {client} from 'src/utils/api'
+import * as api from 'src/client'
```

Comment:
again makes me really nervous to be using two different clients to be interacting with same resource. Let's either switch or not switch

Reply:
I hear your concerns, let me know what you think in the previous comment